### PR TITLE
#4144 new wallet changes

### LINKED ISSUE #4144: wallet connect new design remarks
Hello! small notes with new design:
- no more change account button?
- disconnect is clearing `localStorage` so it will always ask for wallet authentification check
- how can I know wich extension is supported on kodadot since we hide the one that are not installed?
- mobile responsiveness & wallet selection are small:
Screenshot 2022-10-19 at 10-38-30 KodaDot - Kusama NFT Market Explorer

_Originally posted by @roiLeo in https://github.com/kodadot/nft-gallery/issues/4113#issuecomment-1283655495_

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #4144 
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2022-11-01 at 6 50 24 PM

Screen Shot 2022-11-01 at 6 50 13 PM

### CODE REVIEW

**Comment by prachi00** [comment]
For now, I am showing all the wallets but showing the installed one at the top
Also, fixed all the UI/UX fixes mentioned by @exezbcz

**Review by roiLeo — COMMENTED** [review]
Hey!
small notes from issue:
- can we increase size of wallet account item in the list (at least test it with 48px)
- could you try to fix disconnect btn wich is reloading the app/page


Remain:
> how can I know wich extension is supported on kodadot since we hide the one that are not installed?

we can try to display uninstalled wallet in another list like on solana

Screenshot 2022-11-02 at 08-37-11 Solanart - Discover collect and trade NFTs

**Comment by prachi00** [comment]
> small notes from issue:

@exezbcz for 1st, can we increase the height of these? 
Screen Shot 2022-11-02 at 7 07 45 PM

**Comment by prachi00** [comment]
Screen Shot 2022-11-02 at 7 09 41 PM
@roiLeo so on disconnect we clear the session and reload the page, I am not sure if we should be removing this? I can take a look at change account btn though?

**Comment by roiLeo** [comment]
>  on disconnect we clear the session and reload the page, I am not sure if we should be removing this? 

I don't know what this fixes maybe @preschian has an idea but from a user point of view I now find it compelling to change accounts on the platform

**Comment by prachi00** [comment]
> hi, the clear session on the disconnect button initially to anticipate these errors:
> 
> * [Cache (weird) bug on Firefox #3842 (comment)](https://github.com/kodadot/nft-gallery/issues/3842#issuecomment-1256039281)
> * [Error when tryying to create identity on basilisk #4091 (comment)](https://github.com/kodadot/nft-gallery/issues/4091#issuecomment-1274266384)
> 
> previously, the only way for the user to clear the session is from a toast notification. the toast can be easily ignored when the user navigates to the other page. the toast itself will be triggered if there are new deployments. in case there is no new deployment and the users are stuck with an older cache, there is one alternative way to clear the cache, and I put that in the "Logout" button
> 
> does the `clearSession()` block this task?

maybe we can just add change account btn for now?

**Comment by roiLeo** [comment]
> maybe we can just add change account btn for now?

Yep, I would love to see it again

**Comment by yangwao** [comment]
> so on disconnect we clear the session and reload the page,

yes, let's keep this as there are some cache issue with some package which provides networks and I noticed it's not fully fixed as it was working previously

**Comment by prachi00** [comment]
created another issue for change button

File: components/common/WalletModal.vue [hunk 1]
```diff
@@ -259,9 +260,12 @@ export default class WalletModal extends mixins(UseApiMixin, ChainMixin) {
         .dropdown-content {
           box-shadow: none;
           padding-top: 0px;
+          border: none;
           a {
-            padding: 0.375rem 1rem;
-            height: 29px;
+            padding: 5px 32px;
```
Comment (roiLeo) [inline components/common/WalletModal.vue:265]:
```suggestion
```

I would remove this and add `p-2` class on `b-dropdown-item` to increase size

File: components/common/WalletModal.vue [hunk 2]
```diff
@@ -314,18 +318,27 @@ export default class WalletModal extends mixins(UseApiMixin, ChainMixin) {
...
       a {
         border-bottom: 1px solid #000000;
         margin-bottom: 0;
       }
     }
+
+    .wallet-name {
+      padding: 10px 32px;
+      margin-bottom: 0;
+    }
```
Comment (roiLeo) [inline components/common/WalletModal.vue:341]:
```suggestion
```

I would remove this, remove `subtitle` class from wallet-name & add `px-6 py-2`

**Review by roiLeo — COMMENTED** [review]
Waiting for #4144 final design

**Comment by exezbcz** [comment]
> @yangwao @exezbcz have we decided on the final design for this one yet?

not yet, i will have a look on it after 15.11. I think we can leave it for now how it is.

**Comment by yangwao** [comment]
We should probably move the select box to be drop-down under connect menu

**Comment by yangwao** [comment]
we will redesign wallet menu once more time anyway

good for now

pay 50

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 28.17 USD/KSM ~ 1.775 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x7532a654f665c2f8c3febc5f65107727d0a136abb74661aa4009e260a18dba0a](https://kusama.subscan.io/extrinsic/0x7532a654f665c2f8c3febc5f65107727d0a136abb74661aa4009e260a18dba0a)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**